packages/react/src/components/ContentGroup/ContentGroup.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import { settings } from 'carbon-components';
+import { featureFlag } from '@carbon/ibmdotcom-utilities';
+import { DDS_CONTENT_GROUP } from '../../internal/FeatureFlags';
+
+const { stablePrefix } = ddsSettings;
+const { prefix } = settings;
+
+/**
+ * ContentArrayGroup Component, for use with content arrays
+ *
+ * @param {object} props props object
+ * @param {*} props.children JSX Components
+ * @param {object} props.heading  Heading objects
+ * @param {string} props.heading.type Heading style
+ * @param {string} props.heading.copy Heading text
+ * @returns {*} JSX ContentArrayGroup component
+ */
+const ContentGroup = ({
+  children,
+  heading = { type: 'heading-4', ...heading },
+}) => {
+  return featureFlag(
+    DDS_CONTENT_GROUP,
+    <div
+      className={`${prefix}--content-group`}
+      data-autoid={`${stablePrefix}--content-group`}>
+      <div className={`${prefix}--content-group__container`}>
+        <div className={`${prefix}--content-group__col`}>
+          <h3
+            data-autoid={`${stablePrefix}--content-group__title`}
+            className={`${prefix}--content-group__title ${prefix}--content-group__${heading.type ||
+              'heading-4'}`}>
+            {heading.copy}
+          </h3>
+        </div>
+        <div
+          data-autoid={`${stablePrefix}--content-group__children`}
+          className={`${prefix}--content-group__children`}>
+          {children}
+        </div>
+      </div>
```

Comment:
I need to add it to the functional specs, but talking to the Web Simplification team I think we're going to need an optional CTA here at the end as well.

![image](https://user-images.githubusercontent.com/3793636/71526071-94846280-289a-11ea-9b7c-223c9f0098fe.png)

Reply:
Is this going to be another `CardLink`?

Reply:
The only examples I have in front of me indicate it's only a `CardLink`, but based on my understanding of how the CTA section works it might be smart to add the `linkWithIcon` as an optional CTA because it all depends on the hierarchy of the page.

`LinkWithIcon` -> `CardLink` -> `FeaturedLink`.